kube.libsonnet

```diff
@@ -577,6 +577,16 @@
     assert std.length(rel_paths) == 0 : "paths must be absolute: " + rel_paths,
   },
 
+  ManagedCertificates(name): $._Object('networking.gke.io/v1beta1', 'ManagedCertificate', name, domains) {
```

Comment:
Couple things: 
* It should be singular `ManagedCertificate`
* Note that `_Object()` receives 3 arguments, you'll need to handle `domains` via field overriding (as hinted from the example above).